Description: Don't install binarys with a version in them
 We don't need them.
Forwarded: not-needed
Author: Thomas Weber <tweber@debian.org>
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -697,7 +697,7 @@
 	chmod a+rx $@
 endif
 
-install-exec-hook: make-version-links
+install-exec-hook: 
 
 install-data-hook: install-oct
 
